chapters/annotations.tex

```diff
@@ -1777,16 +1776,17 @@ \subsubsection{Conversion rules}\label{conversion-rules}
   number = 10, use_numberPort = not true);
 Modelica.Blocks.Math.LinearDependency linearDep(y0 = 2, k1 = 0, k2 = 2);
 model A
-  import Modelica.Electrical.Machines.BasicMachines;
-  extends BasicMachines.QuasiStaticDCMachines.DC_ElectricalExcited;
+  import My.Library;
+  extends Library.Package.ActualClass;
 end A;
 model B
   extends A;
-  Boolean b = a.quasiStatic;
+  Boolean b = correct;
 end B;
 \end{lstlisting}
-The \lstinline!convertElement! call for \lstinline!DC_ElectricalExcited! is needed to avoid relying on base classes in the original library where \lstinline!DC_ElectricalExcited! inherits from \lstinline!PartialBasicDCMachine!.
+The \lstinline!convertElement! call for \lstinline!ActualClass! is needed to avoid relying on base classes in the original library where \lstinline!ActualClass! inherits from \lstinline!PartialBase!.
 However, the inheritance among the models to convert (in this case \lstinline!B! inherits from \lstinline!A!) should be handled.
+Note that conversion works regardless of the import of \lstinline!My.Library!.
```

Comment:
Unfortunately, I don't have enough experience with conversions to be able to tell whether this is a trivial change or not.  I think someone like @sjoelund would be in a better position to review this.

Reply:
It is not a change (or rather it was changed in https://github.com/modelica/ModelicaSpecification/pull/2522/ ); it just explains why the example has an import-statement.
The previous version had a similar import with a similar result - it was just that it wasn't explained.

Reply:
Yeah, I really have no reason to think this is wrong.  I just reacted to the fact that the new sentence states that this could also be done without the use of an import.  Let's assume it's so trivial that it doesn't need to be discussed further.  Resolving.